plugin/webpack.js

```diff
@@ -6,6 +6,10 @@ var tern = require("../lib/tern");
 require("./commonjs");
 require("./es_modules");
 
+Array.isArray = Array.isArray || function(v) {
```

Comment:
I would like to get this merged, how can I help ? @chemzqm would it be possible for you to fix the comment by @marijnh ? I can do that, but I will probably have to copy your changes to my own branch and issue another pull request